src/components/IncidentStatusTag/index.tsx

```diff
@@ -15,19 +15,17 @@ const statusClsMap = {
 
 export function IncidentStatusTag({ status, size = "md", className }: IProps) {
   return (
-    <button
-      type="button"
+    <div
```

Comment:
Rather than applying the same style in both places, can we not create a new component with just the styling that is re-used ?

Reply:
sure